src/Uno.UI/ts/Windows/Storage/StorageFolder.Native.ts

```diff
@@ -0,0 +1,64 @@
+﻿
+namespace Windows.Storage {
+
+	export class StorageFolderNative {
+		private static _folderMap: Map<string, FileSystemDirectoryHandle> = new Map<string, FileSystemDirectoryHandle>();
+
+		public static AddHandle(guid: string, handle: FileSystemDirectoryHandle) {
+			this._folderMap.set(guid, handle);
+		}
+
+		public static RemoveHandle(guid: string) {
+			this._folderMap.delete(guid);
+		}
+
+		public static GetHandle(guid: string): FileSystemDirectoryHandle {
+			return this._folderMap.get(guid);
+		}
+
+		/**
+		 * Creates a new folder inside another folder.
+		 * @param parentGuid The GUID of the folder to create in.
+		 * @param folderName The name of the new folder.
+		 */
+		public static async CreateFolderAsync(parentGuid: string, folderName: string): Promise<string> {
+			const parentHandle = this.GetHandle(parentGuid);
+
+			const newDirectoryHandle = await parentHandle.getDirectoryHandle(folderName, {
```

Comment:
We should probably provide a fallback/error system when this API is not available. What happens in Firefox ?

Reply:
We would need to use a polyfill, such as [browser-nativefs](https://web.dev/browser-nativefs/). I considered this, but wasn't sure how to go about adding the dependency.

Reply:
Will adjust this as part of other WASM changes, for now let's keep this :-)